xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xsd:schema xmlns="http://www.netex.org.uk/netex" xmlns:netex="http://www.netex.org.uk/netex" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:ifopt="http://www.ifopt.org.uk/ifopt" targetNamespace="http://www.netex.org.uk/netex" elementFormDefault="qualified" attributeFormDefault="unqualified" version="2.0" id="netex_facility_support">
 	<xsd:include schemaLocation="../netex_responsibility/netex_relationship.xsd"/>
+	<xsd:include schemaLocation="../netex_genericFramework/netex_assignment_version.xsd"/>
+	<xsd:include schemaLocation="../netex_reusableComponents/netex_availabilityCondition_version.xsd"/>
+	<xsd:include schemaLocation="../../netex_part_1/part1_tacticalPlanning/netex_servicePattern_support.xsd"/>
+	<xsd:include schemaLocation="../../netex_part_2/part2_journeyTimes/netex_call_support.xsd"/>
```

Comment:
This ( with the link to Calls) creates a reference from Part 1 to Part 2 of NeTEx  (and also a circular reference)
But that's not easy to solve (it would require to have the RestrictedServiceFacilitySetRef here, a specialisation with From/To to SPinJP in the netex_servicePattern_version.xsd and another one refering the Calls in netex_call_version.xsd

Reply:
I already thought that this would come up. Will think about it.

Reply:
I'm not sure of the best way to go from here: may be create a dedicated file, only including the changes from this file : something like _netex_journey_facility.xsd_ and to put somewhere in _../../netex_part_2_ ?